python/servo/command_base.py

```diff
@@ -461,7 +461,7 @@ def build_env(self, is_build=False):
         """Return an extended environment dictionary."""
         env = os.environ.copy()
 
-        if "media-dummy" not in self.features:
+        if "media-gstreamer" in self.features:
```

Comment:
This worked before the change, but now it doesn't because when running test we simply do not provide any features.

Reply:
I think unconditionally running `set_gstreamer_environment_variables_if_necessary` would be best.